src/log.js

```diff
@@ -244,25 +245,22 @@ class Log extends GSet {
     // TODO: move to Entry
     const verifyEntries = async (entries) => {
       const isTrue = e => e === true
-      const getPubKey = e => e.getPublic ? e.getPublic('hex') : e
       const checkAllKeys = (keys, entry) => {
         const keyMatches = e => e === entry.key
         return keys.find(keyMatches)
       }
-      const pubkeys = this._keys.map(getPubKey)
 
       const verify = async (entry) => {
         if (!entry.key) throw new Error("Entry doesn't have a public key")
         if (!entry.sig) throw new Error("Entry doesn't have a signature")
 
-        if (this._keys.length === 1 && this._keys[0] === this._key ) {
-          if (entry.id !== this.id) 
-            throw new Error("Entry doesn't belong in this log (wrong ID)")
+        if (this._keys.length === 1 && this._keys[0] === this._key) {
+          if (entry.id !== this.id) { throw new Error("Entry doesn't belong in this log (wrong ID)") }
```

Comment:
Yep, the thing is that to make sure we don't mess this up we'd have to change standard js config and add this rule:

```
'brace-style': ['error', '1tbs', { allowSingleLine: false }]
```

I feel I should just update the `esling-config-orbitdb` package I've created and put it there since it's likely we'll wanna add more exceptions/make changes to it later anyway

Reply:
BTW, I'd like to include `'max-len': ['error', { code: 80, ignoreComments: true }],` too if you like it :)

https://eslint.org/docs/rules/max-len